android/build.gradle

```diff
@@ -1,10 +1,16 @@
 buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+	// The Android Gradle plugin is only required when opening the android folder stand-alone.
+    // This avoids unnecessary downloads and potential conflicts when the library is included as a
+    // module dependency in an application project.
+    if (project == rootProject) {
+        repositories {
+            google()
+            jcenter()
```

Comment:
I think we should use `mavenCentral()` instead of `jcenter()` as JCenter has been shutdown 

https://blog.gradle.org/jcenter-shutdown

Reply:
Good catch @maheshwarimrinal, You're right, this is an old PR and I completely forget to update it. is it enough to replace `jcenter()`  with `mavenCentral()` or some additional steps is required? (Unfortunately, I don't have time right now to read the toturials about migrating to mavenCentral.

Reply:
This is enough, I've already merged multiple PR's in many repos and they are working fine :)

Reply:
Thanks, I changed the setting to use mavenCentral instead of jcenter